Resources/Prototypes/_EE/Entities/NanoTrasen/Modsuits/engineering-responsory.yml

```
@@ -0,0 +1,180 @@
+          - state: equipped-FEET-sealed
+
+- type: entity
+  id: SuitStorageResponsoryEngineering
+  parent: SuitStorageBase
+  suffix: Engineering Responsory Modsuit
+  components:
+    - type: StorageFill
+      contents:
+        - id: OxygenTankFilled
+        - id: ClothingModsuitResponsoryEngineering
+        - id: ClothingMaskBreath
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)